en/docs/roadmap/language-support.md

```diff
@@ -0,0 +1,4 @@
+# Supported Ballerina language features
+
+This document provides a high-level overview of the [Ballerina](https://ballerina.io) language features that are supported in the current release. The Ballerina language is designed to be a general-purpose programming language that supports a wide range of use cases. 
```

Comment:
We have to highlight here, the low-code features available in the current release. Pro-code features remain as it is.

Reply:
Will add in a separate PR